oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/app.js

```diff
@@ -1,20 +1,26 @@
-import { createFormAppInit, parseFormAppConfig } from "@js/oarepo_ui/forms";
 import FormAppLayout from "./FormAppLayout";
 import {
-  VocabularyFormControlPanel,
-  VocabularyFormFields,
   VocabularyFormFieldsAwards,
   VocabularyFormFieldsNames,
   VocabularyFormFieldsFunders,
   VocabularyFormFieldsAffiliations,
 } from "./components";
+import { DepositFormApp, parseFormAppConfig } from "@js/oarepo_ui/forms";
+import React from "react";
+import { OARepoDepositSerializer } from "@js/oarepo_ui/api";
 
-const { formConfig } = parseFormAppConfig();
-const { overridableIdPrefix } = formConfig;
+import ReactDOM from "react-dom";
+
+const recordSerializer = new OARepoDepositSerializer(
```

Comment:
This serializer used to serializee for RDM record fields - might cause problems with vocabulary fields?